examples/msgpack/src/tests.rs

```diff
@@ -0,0 +1,105 @@
+extern crate rmp_serde;
+
+use rocket;
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::{Status, ContentType};
+use rocket::Response;
+use std::collections::HashMap;
+
+#[derive(Deserialize)]
+#[allow(dead_code)]
```

Comment:
This should be just for the id field. The example returns an id field, but that field is never used in the tests.